frontend/components/Header/Links.js

```diff
@@ -4,14 +4,39 @@ import s from 'styled-components'
 import { Icon } from '../common'
 import { LOGIN_URL } from '../../utils'
 import { mediaMaxWidth, MD } from '../../constants/measurements'
-import { MEDIUM_GRAY, DARK_GRAY, BORDER } from '../../constants/colors'
+import {
+  SNOW,
+  SNOW_ALPHA,
+  CLUBS_RED,
+  MEDIUM_GRAY,
+  DARK_GRAY,
+  BORDER,
+} from '../../constants/colors'
 import { logEvent } from '../../utils/analytics'
 
 const StyledIcon = s(Icon)`
   opacity: 0.5;
   margin-right: 4px;
 `
 
+const LoginButton = s.a`
+  border: 0;
+  background-color: ${CLUBS_RED};
+  padding: 14px 20px;
+  margin: auto;
+  color: ${SNOW} !important;
+
+  &:hover {
+    color: ${SNOW_ALPHA} !important;
+  }
+
+  ${mediaMaxWidth(MD)} {
+    padding: 8px 0;
+    padding-top: .4rem;
+    width: 5rem !important;
```

Comment:
Why is fixing the width necessary? Can't we just use wider padding on the sides?